packages/generator/tsconfig.json

```diff
@@ -6,7 +6,7 @@
     },
     "exclude": [
         "node_modules",
-        "gulpfile.ts",
+        "gulpfiles",
```

Comment:
Is this still valid after the rename?

Reply:
yeah, because this is the name of the directory and it is still the same.